ssas/monitoring/monitoring.go

```diff
@@ -0,0 +1,48 @@
+package monitoring
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/newrelic/go-agent/v3/newrelic"
+)
+
+var a *apm
+
+type apm struct {
+	App *newrelic.Application
+}
+
+func GetMonitor() *apm {
+	if a == nil {
+		target := os.Getenv("DEPLOYMENT_TARGET")
+		if target == "" {
+			target = "local"
+		}
+		app, err := newrelic.NewApplication(
+			newrelic.ConfigAppName(fmt.Sprintf("BCDA-SSAS-%s", target)),
+			newrelic.ConfigLicense(os.Getenv("NEW_RELIC_LICENSE_KEY")),
+			newrelic.ConfigEnabled(true),
+			newrelic.ConfigDistributedTracerEnabled(true),
+			func(cfg *newrelic.Config) {
+				cfg.HighSecurity = true
+			},
+		)
+		if err != nil {
+			log.Println(err)
```

Comment:
oh, I just realized you'll want to use `Logger` (from logging.go) instead of the default `log` instance.

You may need to import `"github.com/CMSgov/bcda-ssas-app/ssas"` in order to do that:

```suggestion
			ssas.Logger.Error(err)
```